modules/aioseop_importer_exporter.php

```diff
@@ -142,96 +272,148 @@ function parse_ini_helper( $array ) {
-		                for($i=0;$i<count($elem);$i++) { 
-		                    $content .= $key2."[] = \"".$elem[$i]."\"\n"; 
-		                } 
-		            } else if ($elem=="") $content .= $key2." = \n"; 
-		            else $content .= $key2." = \"".$elem."\"\n"; 
-		        } 
-		    } 
-		    return $content; 
+
+		/**
+		 * Gets ini file.
+		 * @since 1.0.0
+		 * @param array $assoc_arr, Boolean $has_sections.
+		 * @return array.
+		 */
+		function get_ini_file( $assoc_arr, $has_sections=TRUE ) {
+		    $content = "";
+		    if ( $has_sections ) {
+		        foreach ( $assoc_arr as $key=>$elem ) {
+		            $content .= "[".$key."]\n";
+		            foreach ( $elem as $key2=>$elem2 ) {
+		                if( is_array( $elem2 ) )
+		                {
+		                    for( $i=0; $i<count( $elem2 ); $i++ )
+		                    {
+		                        $content .= $key2."[] = \"".$elem2[$i]."\"\n";
+		                    }
+		                }
+		                else if ( $elem2=="" )
+							$content .= $key2." = \n";
+		                else
+							$content .= $key2." = \"".$elem2."\"\n";
+		            }
+		        }
+		    } else {
+		        foreach ( $assoc_arr as $key=>$elem ) {
+		            if( is_array( $elem ) ) {
+		                for( $i=0; $i<count( $elem ); $i++ ) {
+		                    $content .= $key2."[] = \"".$elem[$i]."\"\n";
+		                }
+		            } else if ( $elem=="" ) $content .= $key2." = \n";
+		            else
+						$content .= $key2." = \"".$elem."\"\n";
+		        }
+		    }
+		    return $content;
 		}
-		
+
+
 		function parse_ini_advanced( $string ) {
-			return $this->recursive_parse( $this->parse_ini_helper( parse_ini_string( $string, true ) ) );	
+			return $this->recursive_parse(
+				$this->parse_ini_helper(
+					parse_ini_string( $string, true )
+				)
+			);
 		}
-		
-		function do_importer_exporter( ) {
-			$submit = null; $count = 0; $post_exists = null; $post_warning = null;
+
+		/**
+		 * Gets ini file.
+		 * @since 1.0.0
+		 * @since 1.0.1 File imported is sanitized and exception handling.
+		 * @global $aioseop_options, $aiosp, $aioseop_module_list.
+		 */
+		function do_importer_exporter() {
+			$submit = null;
+			$count = 0;
+			$post_exists = null;
+			$post_warning = null;
 			global $aioseop_options, $aiosp, $aioseop_module_list;
-			if ( isset( $_REQUEST['nonce-aioseop'] ) ) $nonce = $_REQUEST['nonce-aioseop'];
-			$post_fields = Array( 'keywords', 'description', 'title', 'meta', 'disable', 'disable', 'disable_analytics', 'titleatr', 'menulabel', 'togglekeywords' );
-			if ( !empty( $_FILES['aiosp_importer_exporter_import_submit']['tmp_name'] ) ) $submit = "Import";
-			if ( !empty( $_REQUEST['export_submit'] ) ) $submit = 'Export';
-			if ( ($submit != null ) && ( wp_verify_nonce( $nonce, 'aioseop-nonce') ) ) {
+			if ( isset( $_REQUEST['nonce-aioseop'] ) )
+				$nonce = $_REQUEST['nonce-aioseop'];
+			$post_fields = Array(

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused local variables such as '$post_fields'.](https://www.codacy.com/app/mrtorbert/all-in-one-seo-pack/file/2692638788/issues/source?bid=3283401&fileBranchId=3340169#l379)